lib/spack/spack/package.py

```diff
@@ -1309,7 +1309,6 @@ def do_fetch(self, mirror_only=False):
             tty.debug('No fetch required for {0}: package has no code.'
                       .format(self.name))
 
-        start_time = time.time()
```

Comment:
Previously, "fetch time" would include the time that Spack waited at the prompt for the user to confirm or deny no-checksum/deprecated versions.